src/main/java/frc/robot/subsystems/conveyor/commands/FeedShooter.java

```diff
@@ -0,0 +1,34 @@
+package frc.robot.subsystems.conveyor.commands;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.conveyor.Conveyor;
+
+/**
+ * This command convey the balls to the shooter.
+ */
+public class FeedShooter extends CommandBase {
+    private final Conveyor conveyor;
+    private final double power;
+
+    public FeedShooter(Conveyor conveyor, double power) {
+        this.conveyor = conveyor;
+        this.power = power;
+
+        addRequirements(conveyor);
+    }
+
+    @Override
+    public void execute() {
+        conveyor.setPower(power);
+    }
+
+    @Override
+    public boolean isFinished() {
+        return false;
```

Comment:
Maybe add that if the conveyor is empty than the function should end

Reply:
? Maybe or not maybe?